.github/workflows/main.yaml

```diff
@@ -10,7 +10,7 @@ permissions:
 jobs:
   go-bench:
     runs-on: ubuntu-latest
-    timeout-minutes: 15
+    timeout-minutes: 30 # until we do https://github.com/openfga/openfga/issues/1172
```

Comment:
Cross reference this issue with #1172 so that we remember this 😄

Reply:
done